lib/widgets/info_element.dart

```diff
@@ -16,24 +18,40 @@ class InfoElement extends StatelessWidget {
     this.isLastItem = false,
     this.url,
     this.color,
+    this.copyText,
   });
 
+  Widget buildContentWithCopyIcon(BuildContext context, ThemeData theme) {
+    return Row(
+        mainAxisAlignment: MainAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          buildContent(theme),
+          Flexible(child: CopyButton(copyContent: copyText ?? '')),
+        ]);
+  }
+
+  Widget buildContent(ThemeData theme) {
+    return url != null
+        ? CustomLink(text: text, url: url ?? '', color: color)
+        : Text(text,
+            style: (color != null
+                ? theme.textTheme.bodyLarge?.copyWith(color: color)
+                : theme.textTheme.bodyLarge));
+  }
+
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
     return Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
       Text(
         label,
-        style: plainText
-            ? theme.textTheme.bodyLarge
-            : theme.textTheme.displaySmall,
+        style:
```

Comment:
The title should be bold
![Screenshot 2024-10-07 at 15 58 08](https://github.com/user-attachments/assets/0b6753e0-ad61-4661-80da-521849a33ee2)